ksql-engine/src/test/java/io/confluent/ksql/services/KafkaTopicClientImplTest.java

```diff
@@ -290,8 +315,24 @@ public void shouldThrowTopicDeletionDisabledException()
     kafkaTopicClient.deleteTopics(Collections.singletonList(topicName1));
   }
 
+  @Test(expected = TopicAuthorizationException.class)
+  public void shouldFailToDeleteThrowTopicAuthorizationException()
+          throws InterruptedException, ExecutionException, TimeoutException {
+    // Given:
+    expect(adminClient.deleteTopics(anyObject())).andReturn(
+      (deleteTopicException(new TopicAuthorizationException(
+          AclErrorMessageUtil.generateErrorMessage(AclOperation.DELETE, topicName1)
+      ))));
+    replay(adminClient);
+
```

Comment:
Let's use `expectedException.expectMessage(...)` to verify the message contained in the `TopicAuthorizationException`, as that's the crux of changes in this PR :)